draft-ietf-mls-protocol.md

```diff
@@ -2066,7 +2036,7 @@ enum {
     add(1),
     update(2),
     remove(3),
-    presharedkey(4)
+    psk(4)
```

Comment:
@bifurcation wanted to have this spelled out. I'm happy with the abbreviation.

Reply:
I can live with this.

Reply:
Or `pre_shared_key(4)`